src/Stratis.Bitcoin/P2P/PeerConnector.cs

```diff
@@ -173,14 +169,20 @@ internal bool IsPeerConnected(IPEndPoint endPoint)
         }
 
         /// <inheritdoc/>
-        public void StartConnectAsync()
+        public void StartConnectAsync(NetworkPeerConnectionParameters parameters, PeerAddressManagerBehaviourMode mode)
         {
+            this.parentParameters = parameters;
+
```

Comment:
I wonder if we want to move the 
```
clonedParameters.TemplateBehaviors.Add(new ConnectionManagerBehavior(false, this, this.loggerFactory));
```
From the ConnectionManager in to each connector.

Reply:
Yes this will be done in next PR 👍